tdp_vars_defaults/livy-spark3/livy-spark3.yml

```diff
@@ -60,6 +60,22 @@ livy_conf:
   livy.superusers: "knox,hue,livy"
   livy.server.access-control.enabled: "true"
 
```

Comment:
Maybe add a commentary for jmx_exporter variables (as for livy.yml):
```# jmx exporter config```

Reply:
Comment added